app/controllers/home_controller.rb

```diff
@@ -63,6 +63,20 @@ def research
     end
   end
 
+  # Display all posts assuming all posts == activity
+  def all_posts
+    if current_user
+      # underscore used for variables not needed
+      # I'm assuming all comments == @comments + @answer_comments
+      # TO DO: Identify what will be pinned at the top of the page. A featured post maybe?
+      _, _, @notes, @wikis, _, @comments, @answer_comments = activity
```

Comment:
I think this function is being used some other place also and supports pagination also which is great.

Reply:
This is where we only need to show "notes" for the initial view, since unlike the current old dashboard we won't be mixing them together. We'll make separate routes, URLs , and tabs for each type. 

Would this perhaps best fit into... a new controller? I'm not sure. If the home controller isn't too full, i guess it's fine here. Your choice! Thank you!!!